### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by germartinez**
## Context

This PR:
- Updates the README file content
- Fixes some broken links (not really broken) highlighted by the linkcheker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,84 @@
-# <img src="assets/svg/safe-logo-white.svg" height="60" valign="middle" alt="Safe{Docs}" />
+# Safe Documentation
```
Comment (germartinez) [inline README.md:1]: Note: This image is not visible in light mode. Removing it for now.

**Comment by germartinez** [comment]
Note: The previous image was not visible in light mode. Removing it for now.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,84 @@
...
 
 [![License](https://img.shields.io/github/license/safe-global/safe-docs)](https://github.com/safe-global/safe-docs/blob/main/LICENSE.md)
 ![GitHub package.json version (branch)](https://img.shields.io/github/package-json/v/safe-global/safe-docs)
-<!-- ![Tests](https://img.shields.io/github/actions/workflow/status/safe-global/safe-docs/test.yml?branch=main&label=tests) -->
 
-# Safe{Docs}
+This repository hosts [Safe](https://safe.global) documentation.
 
-This repository hosts [Safe](https://safe.global)'s documentation website, built with [Nextra](https://nextra.site). Live version can be found at [docs.safe.global](https://docs.safe.global).
+The project is built using [Nextra](https://nextra.site) framework and it can be found live at [docs.safe.global](https://docs.safe.global).
```
Comment (tanay1337) [inline README.md:8]:
```suggestion
The documentation is built with [Nextra](https://nextra.site/) and is live at [docs.safe.global](https://docs.safe.global/).
```

File: README.md [hunk 3]
```diff
@@ -1,36 +1,84 @@
...
 
-## Quick Start
+## Installation
 
-First, run `pnpm i` to install the dependencies.
+Install the project:
```
Comment (tanay1337) [inline README.md:12]:
```suggestion
Install the dependencies using [pnpm](https://pnpm.io/):
```

File: README.md [hunk 4]
```diff
@@ -1,36 +1,84 @@
...
+pnpm linkcheck
+```
+
+## Execution
+
+The project can be run with a server that's executed in development and production mode.
+
+### Development mode
+
+Run the server in development mode:
```
Comment (tanay1337) [inline README.md:38]:
```suggestion
Run the server in development mode using the following command:
```

File: README.md [hunk 5]
```diff
@@ -1,36 +1,84 @@
...
+
+### Production mode
+
+Build the project:
+
+```
+pnpm build
+```
+
+Run de server in production mode:
```
Comment (tanay1337) [inline README.md:52]:
```suggestion
Run the server in production mode using the following command:
```

File: README.md [hunk 6]
```diff
@@ -1,36 +1,84 @@
...
+```
+pnpm start
+```
+
+## Testing
+
+Create an environment file in the root of the project and copy the content from the `.env.example` file:
+
+```
+touch .env
```
Comment (tanay1337) [inline README.md:63]:
```suggestion
cp .env.example .env
```

File: README.md [hunk 7]
```diff
@@ -1,36 +1,84 @@
...
+
+Run de server in production mode:
+
+```
+pnpm start
+```
+
+## Testing
+
+Create an environment file in the root of the project and copy the content from the `.env.example` file:
```
Comment (tanay1337) [inline README.md:60]:
```suggestion
Create an environment file in the root of the project and copy the content from the `.env.example` file using the following command:
```

File: README.md [hunk 8]
```diff
@@ -1,36 +1,84 @@
...
+
+```
+touch .env
+```
+
+```
```
Comment (tanay1337) [inline README.md:66]: The following code snippet can go and instead we can add a note reminding them to update the environment variables.

File: README.md [hunk 9]
```diff
@@ -1,36 +1,84 @@
...
+NEXT_PUBLIC_HOST_URL='http://localhost:3000'
+NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID=
+NEXT_PUBLIC_GOOGLE_ANALYTICS_DOMAIN=
+```
+
+Run the tests:
```
Comment (tanay1337) [inline README.md:72]:
```suggestion
Run the tests using the following command:
```

**Review by tanay1337 — CHANGES_REQUESTED** [review]
Thanks, @germartinez

File: README.md [hunk 10]
```diff
@@ -1,36 +1,84 @@
...
+
+```
+touch .env
+```
+
+```
+NEXT_PUBLIC_HOST_URL='http://localhost:3000'
+NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID=
+NEXT_PUBLIC_GOOGLE_ANALYTICS_DOMAIN=
+```
```
Comment (germartinez) [inline README.md:70]:
```suggestion
Remember to update the environment variables once the `.env` file is created.
```

**Review by tanay1337 — APPROVED** [review]

**Pull request merged**